packages/omezarr/src/sliceview/loader.ts

```diff
@@ -94,15 +94,26 @@ export function getVisibleTiles(
         screenSize: vec2;
     },
     plane: CartesianPlane,
-    orthoVal: number,
+    planeLocation:
```

Comment:
the meat of the change - orthoVal was always assumed to be the voxel-index of a slice along the given plane. I renamed it to be more descriptive, and it is now a union to support either index or parameter (percentage) specification